v3.10/getting-started/kubernetes/managed-public-cloud/eks.md

```diff
@@ -18,6 +18,7 @@ To enable Calico network policy enforcement, follow these step-by-step instructi
 
 ### Above and beyond
 
+- [Everything you need to know about Kubernetes pod networking on AWS](https://www.projectcalico.org/everything-you-need-to-know-about-kubernetes-pod-networking-on-aws/)
```

Comment:
@lxpollitt this looks fine but this will only impact the v3.10 release, and I'm guessing you also want this in newer releases as well.

To do that, you need to add this link to the `master/getting-started/kubernetes/managed-public-cloud/eks.md` document as well.